src/server/bnetserver/Realms/WorldListener.cpp

```diff
@@ -65,7 +68,7 @@ void WorldListener::HandleOpen()
 {
     try
     {
-        _worldSocket->bind(std::string("tcp://*:") + std::to_string(_worldListenPort));
+        _worldSocket->bind(std::string("tcp://"+_bindIp+":") + std::to_string(_worldListenPort));
```

Comment:
This simply makes sure that bindIp is honoured.  The old code simply bound to all addresses which is not good on a multihomed server.  Binding always to loopback is also a solution provided the IPC is always local.

It's possible that particular change belongs in its own pull request as it's not directly related to ipv6 - I just noticed it when I was going through the various bits of networking code.